quesma/quesma/ui/html_utils.go

```diff
@@ -81,9 +81,9 @@ func (qmc *QuesmaManagementConsole) generateTopNavigation(target string) []byte
 	if target == "ab-testing-dashboard" {
 		buffer.Html(` class="active"`)
 	}
-	buffer.Html(`><a href="`)
+	buffer.Html(`><a title="Compatibility Report" href="`)
 	buffer.Html(abTestingPath)
-	buffer.Html(`">A/B</a></li>`)
+	buffer.Html(`">CR</a></li>`)
```

Comment:
Don't forget that there are instance of "A/B testing" still left in `ab_testing.go` (such as `A/B Testing - Panel Details`).

It's OK if you want to do it in a followup PR.